cli/src/debugger.rs

```diff
@@ -99,6 +109,24 @@ impl DebugCli {
             },
         });
 
+        cli.add_command(Command {
+            name: "write",
+            help_text: "Write 32bit value to memory",
+
+            function: |cli_data, args| {
+                let address_str = args.get(0).ok_or(CliError::MissingArgument)?;
```

Comment:
It is mandatory :wink:, but we have to handle the case when someone doesn't enter it.
This is done the same as in the other commands, and not really changed in this commit.

I agree that it is not an optimal solution, but fixing it is not a high priority for me, as I see the cli currently more as a kind of debugging tool, and not a polished end-user tool.
It would like to avoid adding even more changes to this PR, and fix this in a different one.